CHANGELOG.md

```diff
@@ -1,18 +1,25 @@
-# v4.0 - mixed domains and versions
+# v4.0
 ## New Features
 ### Major
+- Added support for mixed domains and versions. Layers can be opened with different ATT&CK versions and now support custom domains. See issues [#180](https://github.com/mitre-attack/attack-navigator/issues/180) and [#182](https://github.com/mitre-attack/attack-navigator/issues/182).
+    - Users can specify the ATT&CK version and domain for each layer. A layer with no specified ATT&CK version will default to the current version.
+    - Updated "create new layer" interface to provide access to previous versions of ATT&CK.
+    - Added ability to upgrade an uploaded layer to the current version of ATT&CK.
+    - Updated "create layer from other layers" interface to restrict layer operations to layers of the same domain and version.
+    - Updated configuration file to support dynamic domains and versions. Users can update this file to display multiple domains in a single instance (see _Loading content from local files_ in [the readme](README.md) for more details). See issue [#183](https://github.com/mitre-attack/attack-navigator/issues/183).
```

Comment:
I think we probably also want a Configuration File Format Changes section. I realize the new format is documented in the README, but we also need to provide information about what specifically _changed_ so that people can determine what they need to patch if they had previously customized the config. 

This sort of documentation been done previously in versions [3.0](https://github.com/mitre-attack/attack-navigator/blob/master/CHANGELOG.md#config-file-format-changes) and 2.2.